tests/classification/test_f_beta.py

```diff
@@ -129,7 +130,7 @@ def test_zero_division(metric_class, metric_fn):
     """ Test that zero_division works correctly (currently should just set to 0). """
 
     preds = torch.tensor([1, 2, 1, 1])
-    target = torch.tensor([2, 1, 2, 1])
+    target = torch.tensor([2, 0, 2, 1])
```

Comment:
why is this needed?

Reply:
If there is no `0` class in both the preds and the targets, the new implementation will return `nan` and not 0 as expected by this test. However, to test whether `zero_division` works correctly, all that is needed is 0 `TPs` and 0 `FPs` and >0 `FNs` (see [here](https://github.com/AnselmC/metrics/blob/22c3709c6d4e8a5d6f36c9d410f680e297548610/torchmetrics/functional/classification/f_beta.py#L44)).